src/util/Conversions.h

```diff
@@ -568,10 +571,37 @@ inline string removeLeadingZeros(const string& orig) {
 }
 
 // _____________________________________________________________________________
-bool isXsdValue(const string val) {
+bool isXsdValue(const string& val) {
   // starting the search for "^^ at position 1 makes sure it's not in the
   // quotes as we already checked that the first char is the first quote
-  return val.size() > 0 && val[0] == '\"' &&
-         val.find("\"^^", 1) != string::npos;
+  return !val.empty() && val[0] == '\"' && val.find("\"^^", 1) != string::npos;
+}
+
+// _____________________________________________________________________________
+bool isNumeric(const string& val) {
+  if (val.empty()) {
+    return false;
+  }
+  size_t start = (val[0] == '-') ? 1 : 0;
```

Comment:
I'd guess that `+` is also a valid sign (at least in Turtle "+253" is also a valid integer etc. and I guess this should also be true for sparql queries.

Reply:
Good point